configuration-sample/ods-core.env.sample

```diff
@@ -298,6 +298,9 @@ PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=false
 # List of jira project templates
 JIRA_PROJECT_TEMPLATE_KEY_NAMES=${project.template.default.key},kanban
 
+# List of default groups to be permissioned in openshift on project creation (value is optional and can be empty)
```

Comment:
@stitakis Could you include a comment line with an example of how it would be used, please?

Reply:
something like this?
```
#DEFAULT_OPENSHIFT_PERMISSION_GROUPS="ADMINGROUP=MY-DEFAULT-CLUSTER-ADMIN-GROUP"
```

Reply:
That's perfect, thanks @stitakis